bylaws/008-psr-evolution.md

```diff
@@ -19,7 +19,7 @@ A new minor release of a PHP-FIG package containing interfaces for a PSR MUST fo
  * the PHP version constraint of the PHP-FIG package MUST NOT be altered to use newer language features that would create cross-compatibility issues.
  
 A new major release of a PHP-FIG package containing interfaces for a PSR MUST follow the same rules, with this exception:
- * the new major version of the package MAY contain breaking changes if the implementing packages have a reasonable upgrading path, like the possibility of releasing a cross-compatible implementation with the previous releases.
+ * the new major version of the package MAY contain breaking changes if the implementing packages have a reasonable upgrade path, like the possibility of releasing a cross-compatible implementation with the previous releases;
  * the new major version of the package MAY refer to a new, superseding PSR.
```

Comment:
What's the intent here?  Would this be, say, psr-21, which is a new PHP 8-centric version of Request/Response messages, which then uses the same namespace as PSR-7?

Reply:
That sounds like a question more directly for @Jean85 's PR, rather than mine.  I'm only doing some English touch-ups of his via my PR.

Reply:
Responding to @Crell, yes it would. Obviously that would mean that the PHP-FIG vision would see that as an upgrade path of PSR-7/15/17/18